lib/src/model.dart

```diff
@@ -4674,3 +4416,20 @@ class TypeParameter extends ModelElement {
   @override
   String toString() => element.name;
 }
+
+class _LocatableStrings {
```

Comment:
I appreciate wanting to get rid of the duct tape, but I would prefer we modify Locatable's elementLocation to simply return what warnable needs (I don't think we use elementLocation for any other purposes).

Reply:
Fixed.